packages/config-plugins/src/ios/Paths.ts

```diff
@@ -253,27 +253,6 @@ export function getAllEntitlementsPaths(projectRoot: string): string[] {
   return paths;
 }
 
-/**
- * Get the entitlements file path if it exists.
- *
- * @param projectRoot
- */
-export function getEntitlementsPath(projectRoot: string): string | null {
```

Comment:
This is a breaking change, please keep it here.

Reply:
Should I keep the old implementation or reexport `Entitlements.getEntitlementsPath` here?

If this is used in users config plugins, do we have any method of deprecating it?
If this is used somewhere else in our code, can you point where? I would like to also update it.

Reply:
What do you mean? Do you think that there could be some other libs actually using it?

Reply:
Ya this package has very high usage so we shouldn't break anything that is publicly exposed without first going through a deprecation period.